### PR TITLE
use new png images for architecture diagrams

### PR DESCRIPTION
**Pull request by smola**

### CODE REVIEW

File: architecture.md [hunk 1]
```diff
@@ -34,13 +27,7 @@ The entry point of the container is the Go UAST normalizer, which communicates
...
 looks like:
 
-```mermaid
-graph LR
-    Server-- stdin/stdout<br/>gRPC -->UAST
-    subgraph Container
-        UAST["UAST Normalizer<br/>(Go)"]-- stdin/stdout<br/>JSON -->AST["Code Parser<br/>(Python)"]
-    end
-```
+![Driver Architecture](./images/architecture-driver.png)
```
Comment (alcortesm) [inline architecture.md]: What do the arrows in the pictures mean?

If they mean data communication, then maybe we should draw heads at both ends.

I think this also apply to the other picture.

Comment (smola) [inline architecture.md]: The link means communication and the direction indicates who initiates communication and make requests.

Comment (alcortesm) [inline architecture.md]: Maybe you should explain what the arrows mean in the caption of the image or on the text.

**Comment by smola** [comment]
@alcortesm I have updated to a single diagram, which includes your proposed change.

**Comment by alcortesm** [comment]
![https://media.giphy.com/media/l2Sq5GffrCyUMEXjW/giphy.gif]

**Pull request merged**